wagtail/wagtailadmin/templates/wagtailadmin/chooser/_browse_results.html

```diff
@@ -4,7 +4,7 @@
 {% include "wagtailadmin/shared/breadcrumb.html" with page=parent_page choosing=1 %}
 
 {% if pages %}
-    {% include "wagtailadmin/pages/listing/_list_choose.html" with allow_navigation=1 orderable=0 pages=pages parent_page=parent_page %}
+    {% include "wagtailadmin/pages/listing/_list_choose.html" with allow_navigation=1 orderable=0 choosing=1 pages=pages parent_page=parent_page %}
```

Comment:
Why is `choosing=1` needed here? `_list_choose.html` doesn't appear to use it (and in any case it would be redundant, because `_list_choose.html` should by definition *only* be used for choosing...)